projects/linuxkit/linuxkit/dependencies.yaml

```diff
@@ -0,0 +1,6 @@
+DHCPCD:
```

Comment:
This is kind of a new pattern, we may want to break these builds out later on, for now we are only going to use them here to produce images we only use during hook build time.  I did model this a bit off how we do the builder-base versions.yaml.  I was thinking this would be easy to add the upgrade script to check for new versions and update.